### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by j1george**
Implemented several UI changes. Let me know what you think

### CODE REVIEW

**Comment by j1george** [comment]
hello

**Comment by j1george** [comment]
@codistryNhan this is a message

**Comment by codistryNhan** [comment]
Hello jorge

File: src/components/game/Game.css [hunk 1]
```diff
@@ -1,3 +1,6 @@
 .game-container {
-  background: #1b2e87;
+  background-color: var(--secondary-blue);
+  height: 100%;
+  padding: 1rem 2rem;
+  padding-top: 5rem;
```
Comment (codistryNhan) [inline src/components/game/Game.css]: Secondary Typo

File: src/components/header/header.css [hunk 2]
```diff
@@ -1,7 +1,28 @@
-header.main-header{
+header.main-header {
   padding: 1rem 1rem 0.5rem 1rem;
+  position:absolute; top:0px; left:0px; right:0px;overflow:hidden;
```
[the remark this answers is not in the document]
Comment (j1george) [reply src/components/header/header.css]: part of https://github.com/codistryNhan/typer/pull/1/files#r349444328

File: src/components/high-scores/HighScores.css [hunk 3]
```diff
@@ -7,6 +7,10 @@
 
 .high-scores-container {
   background: var(--main-blue-dark);
+  padding-top: 5rem;
+
+  display: block;
+  overflow: auto;
```
[the remark this answers is not in the document]
Comment (j1george) [reply src/components/high-scores/HighScores.css]: part of https://github.com/codistryNhan/typer/pull/1/files#r349444328

**Comment by j1george** [comment]
> 
> 
> Fix secondary typo

fixed

**Pull request merged**